sdk/src/main/scala/com/stratio/sparkta/sdk/AggregateOperations.scala

```diff
@@ -50,63 +50,61 @@ object AggregateOperations {
                idCalculated: Boolean,
                dateType: TypeOp.Value): (Option[String], Row) = {
     val timeDimension = dimensionValuesT.timeDimension
+    val filtered = dimensionValuesT.dimensionValues.filter(cube => (cube.dimension.name == timeDimension)).nonEmpty
     val dimensionValuesFiltered =
       filterDimensionValuesByName(dimensionValuesT.dimensionValues,
-      if (timeDimension.isEmpty) None else Some(timeDimension))
-
-    val namesDim = dimensionValuesNames(dimensionValuesFiltered.sorted)
+        if (timeDimension.isEmpty) None else Some(timeDimension))
 
+    val namesDim = dimensionValuesT.cube
     val (valuesDim, valuesAgg) = toSeq(dimensionValuesFiltered, aggregations ++ fixedAggregation)
 
+    val timeDimensionValue = filtered match {
+      case true => Seq(dimensionValuesT.time)
+      case false => Seq(DateOperations.millisToTimeStamp(dimensionValuesT.time))
+    }
+
```

Comment:
If you rebase from master, the time.value calculation is new in aggregateOperations and SchemaFactory:

Mantain the new calculation:

Seq(getTimeFromDateType(dimensionValuesT.time, dateType))